### ISSUE TITLE
add CI regression test for all the GraphQL queries

### ISSUE THREAD

**Issue by Rdataflow** [visualize-admin/visualization-tool#1275]
**Is your enhancement related to a problem? Please describe.**
Currently broken query functions are not detected due to missing CI regression tests on them.

**Describe the solution you'd like**
CI regression tests detecting any broken query functionality.

**Describe alternatives you've considered**
more manual tests :upside_down_face: 

**Usecases and impact**
see broken query function like #1274

**Comment by Rdataflow** [comment]
@bprusinowski I guess this is covered to a certain degree with the k6 tests, right?

**Comment by ptbrowne** [comment]
Thanks for adding the issue Thomas, we could definitely do that. One thing I see that could be a challenge would be the fact that some cubes are changing from time to time, and we already see that with current end to end tests where we have to periodically update the tests since cubes have evolved. 

- Do you think it would be a good idea / feasible to have some tests cube where the data does not change ?

While writing this, I figured that we could use the cube versioning to only query a particular version so that we are confident that values will not change.

**Comment by Rdataflow** [comment]
yes, it would be the best solution I can imagine to pin a version of a cube :+1:
